### PR TITLE
Moth Food Returns: A Taste of the Fleet

### PR DESCRIPTION
**Pull request by EOBGames**
## About The Pull Request
image
Well, I figure it's time to make good on that promise.
In this thrilling sequel to #59412, Moths get the culinary overhaul they've needed for a while.
Adds somewhere in the region of 50 new foods, all tied together by a connection to Moths and their culture, as well as around 10 new drinks.
image
As with #59412, this PR has three aims:
1. Introduce moth culture (as it exists in Common Core lore) to the game in a tangible way.
2. Give further variety to the chef and bartender, including some form of new cooking challenge for them.
3. To give some flavour (metaphorically and literally).

image
This time, I actually will include a complete list of the new foods and their recipes: https://hackmd.io/tFr8ef46TeeF5HkDzxxIhw?view

As a nomadic people, moths have a wide variety of foods from various cultures. In particular, they've adopted a lot from humanity. Expect a mix of stranger and more distinctly alien foods, along with some human classics that have underwent a mothic transformation. Additionally, any language fans may enjoy the comments on each Mothic named food item, giving a glimpse into Mothic as a language.
image
Töchtaüse Berries
image
A popular food on the Mothic Fleet, töchtaüse berries are well known for being deeply unpleasant while raw: they're spicy, bitter, and filled with a mild poison that causes itching. Once heated to a sufficient temperature and mixed with sugar, however, they're a complex flavour that adds nicely to both sweet and savoury dishes.

Psst. Hey. Secret extra feature for secret fans:
image
This charming poster, originally by Capsandi, from #51867. Moffuchi's may be gone, but it's not forgotten.

## Why It's Good For The Game
The lizard food PR was a good first step in making Common Core lore better integrated with the game, but it was only that: a start. This is a continuation that should keep that trend going and making the universe deeper.

## Changelog

:cl:
add: The release of the new cookbook, "Mothic Cuisine: of Fleet and Flotilla" has brought Mothic food to the masses! Try out some new treats, like cheesecake balls, cheese soup, and five-cheese pizza at the kitchen today! There might even be some foods that don't have cheese, too!
add: To coincide with the new popularity of Mothic cuisine, Nanotrasen now stocks seeds for Töchtaüse Berries, a common ingredient in moth cooking, in the Megaseed vendors. Additionally, herbs and bell peppers have also made their way to the station's stock. Botanists, you know what to do.
/:cl:

### CODE REVIEW

**Comment by EOBGames** [comment]
And of course it's conflicting immediately, for fucks sake I can't escape them with this content

**Comment by Googles-Hands** [comment]
Bro didn't you hear, we aren't supposed to add more bloat clothes, I know you like moths but eh

**Comment by Farquaar** [comment]
No cloth food?

**Comment by optimumtact** [comment]
handsclaspedglasses

File: code/game/objects/items/food/misc.dm [hunk 1]
```diff
@@ -650,6 +650,25 @@
 	tastes = list("peaches" = 1, "tin" = 7)
 	venue_value = FOOD_EXOTIC
 
+/obj/item/food/canned/tomatoes
+	name = "canned San Marzano tomatoes"
+	desc = "A can of premium San Marzano tomatoes, from the hills of Southern Italy."
```
Comment (CRITAWAKETS) [inline code/game/objects/items/food/misc.dm:655]: Should be something other than a real location, like Space Italy.

File: code/game/objects/items/food/moth.dm [hunk 2]
```diff
@@ -0,0 +1,884 @@
...
+	desc = "Delicious, creamy, and cheesy, all in one simple package."
+	icon = 'icons/obj/food/food.dmi'
+	icon_state = "mozzarella"
+	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 3)
+	tastes = list("mozzarella" = 1)
+	foodtypes = DAIRY
+	w_class = WEIGHT_CLASS_SMALL
+
+//Base ingredients and miscellany, generally not served on their own
+/obj/item/food/grilled_cheese
```
Comment (CRITAWAKETS) [inline code/game/objects/items/food/moth.dm:77]: wait dont we already have grilled cheese?

**Comment by CRITAWAKETS** [comment]
Items like mozzarella cheese, tomato/pesto sauce and the likes should probably be somewhere more towards what they are than the moth food file.

File: code/game/objects/items/food/misc.dm (hunk 1 above)
Comment (EOBGames) [reply code/game/objects/items/food/misc.dm:655]: Not everything needs to be from space, especially not something with a DOP.

File: code/game/objects/items/food/moth.dm (hunk 2 above)
Comment (EOBGames) [reply code/game/objects/items/food/moth.dm:77]: This is grilled cheese as in a grilled piece of cheese, a la halloumi, not a grilled cheese sandwich. The description quite clearly states that, I think.

Comment (CRITAWAKETS) [reply code/game/objects/items/food/moth.dm:77]: oh im dumb, carry on

**Comment by carshalash** [comment]
Told you that you'd get it man. I knew you could do it eob chan.

File: code/game/objects/items/food/moth.dm (hunk 2 above)
Comment (EricZilla) [reply code/game/objects/items/food/moth.dm:16]: You bout to add moth halloumi???

**Comment by Zytolg** [comment]
POUTINE RETURNS

**Comment by EOBGames** [comment]
I'm going to draft this since there's some stuff needing changed around- it shouldn't take long but I don't want to risk this getting merged while it's not done

**Comment by EOBGames** [comment]
And with that I believe this should be ready for review.

**Review by Kylerace — APPROVED** [review]
code is good, your life is in the spritetainers hands

**Review by Imaginos16 — APPROVED** [review]
Sprites are good, your life is now on the maintainers ha- Oh wait, they already approved!

**Comment by tuckerorchistra** [comment]
Why would moths cook regular food if they eat cloth.

**Comment by Mothblocks** [comment]
why do you cook regular food if you eat bread

**Pull request merged**